opm/grid/CpGrid.hpp

```diff
@@ -398,6 +398,9 @@ namespace Dune
         /// @brief Returns either data_ or distributed_data_(if non empty).
         const std::vector<std::shared_ptr<Dune::cpgrid::CpGridData>>& chooseData() const;
 
+        /// @brief Returns either data_ or distributed_data_(if non empty).
+        std::vector<std::shared_ptr<Dune::cpgrid::CpGridData>>& getData();
+
```

Comment:
Can we stick to one name for the const and mutable function, please? (Either chooseData or getData or somthing even better)

Reply:
Tiny improvement of the name: now both are called "currentData"